draft-irtf-cfrg-vdaf.md

```diff
@@ -3588,19 +3595,21 @@ def verifier_len(self) -> int:
 
 ### Generating the Proof {#flp-bbcggi19-construction-prove}
 
-~~~~
-+------------------+
-| Prove            |
-| +-------------+  |
-| | Valid       |<---- meas
-| | +--------+  |<---- joint rand
-| | | Gadget |  |  |<- prove rand
-| | +--------+  |  |
-| +-------------+  |
-+------------------+
- |
- V proof
-~~~~
+~~~ aasvg
++--------------------+
+| prove()            |
+|  +-------------+   |
+|  | Valid        |<----- meas
+|  |  +--------+  |<----- joint rand
+|  |  | Gadget |  |  |<-- prove rand
+|  |  +-------+   |  |
+|  |              |  |
+|  +--------------+  |
+|                    |
++-+-----------------+
```

Comment:
These aren't quite aligned, and turned into rounded corners
```suggestion
|  +--------------+  |
|  | Valid        |<----- meas
|  |  +--------+  |<----- joint rand
|  |  | Gadget |  |  |<-- prove rand
|  |  +--------+  |  |
|  |              |  |
|  +--------------+  |
|                    |
+-+------------------+
```

Reply:
I was being artistic, lol. I'm fine making these square.